scaffolding-chef-infra/tests/user-windows-default/tests/test.ps1

```diff
@@ -0,0 +1,36 @@
+param (
+    [Parameter()]
+    [string]$PackageIdentifier = $(throw "Usage: test.ps1 [test_pkg_ident] e.g. test.ps1 ci/user-windows/1.0.0/20190812103929"),
+    [string]$PackageSource = $(throw "Usage: test.ps1 [test_pkg_source] e.g. test.ps1 ./results/ci-user-windows-1.0.0-20190812103929-x86_64-windows.hart")
+)
+
+if (-Not (Get-Module -ListAvailable -Name Pester)){
+    hab pkg install core/pester
+    Import-Module "$(hab pkg path core/pester)\module\pester.psd1"
+}
+
+# Install the package
+hab pkg install $PackageSource
+
+Get-Process hab-sup -ErrorAction SilentlyContinue | Stop-Process
+Get-Process hab-launch -ErrorAction SilentlyContinue | Stop-Process
```

Comment:
this works but just an FYI: killing the launcher will kill the supervisor too. AND in the next release `hab sup term` will work on windows!

Reply:
Awesome.

Reply:
I'm going to add an issue for tracking this.

Reply:
#195